### ISSUE TITLE
Android app on FDroid

### ISSUE THREAD

**Issue by gappleto97** [keybase/client#8831]
That would enable a lot of users who try for OSS only to use your app, myself included.

**Comment by Agorise** [comment]
Yeah, this would be awesome since afaik google play app store is blocked in China. 
F-Droid is awesome: https://f-droid.org 
 
Fyi, other good places to post the Keybase apk would be: 
http://www.aptoide.com 
http://slideme.org 
https://alternativeto.net 
 
It's totally free to get the Keybase app published on those sites too :)

**Comment by PC-Admin** [comment]
Yes please, everyone recommends keybase but it doesn't have an FDroid version available. :( Meanwhile your competitor 'Pretty Easy Privacy' has had a client on FDroid for over a year.

**Comment by Serkan-devel** [comment]
Waiting for the Fdroid release of keybase.

**Comment by solvaholic** [comment]
Duplicate of https://github.com/keybase/client/issues/6899

**Comment by Atavic** [comment]
Triplicated as: #9653
